app/controllers/notifications_controller.rb

```diff
@@ -0,0 +1,10 @@
+class NotificationsController < ActionController::API
+  def destroy
+    @notification = Notification.find(params["id"])
+
+    return unless @notification.user_id == current_user.id
+
+    @notification.display = false
```

Comment:
I think we should add a comment here why we dont delete the notification for good and instead set `    @notification.display = false`.